lib/fennec/auth.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Fennec.Auth do
+  @moduledoc false
+  # This module implements authentication and authorization of
+  # the STUN and TURN protocol.
+
+  alias Jerboa.Format.Body.Attribute
+  alias Jerboa.Format.Body.Attribute.{Username, Nonce, Realm}
+  alias Jerboa.Params
+
+  @nonce_bytes 48
+  @nonce_lifetime_seconds 60 * 60 # 1h
```

Comment:
I didn't see anywhere in RFC the "recommended" value for that (although I didn't look for it). The 1h is mentioned as an example in the RFC.